### ISSUE TITLE
PHP 7.3 support will get dropped - needs documentation

### ISSUE THREAD

**Issue by mmattel** [owncloud/docs-server#661]
## WHAT Needs to be Documented?
Removal of PHP7.3 support

## WHERE Does This Need To Be Documented (Link)?
https://doc.owncloud.com/server/next/admin_manual/installation/system_requirements.html
https://doc.owncloud.com/server/next/admin_manual/installation/manual_installation/manual_installation_prerequisites.html

## WHY Should This Change Be Made?
References: https://github.com/owncloud/core/pull/40394 ([full-ci] Drop PHP 7.3)

## (Optional) What Type Of Content Change Is This? 
- [ ] New Content Addition
- [ ] Old Content Deprecation
- [ ] Existing Content Simplification
- [x] Bug Fix to Existing Content

## (Optional) Which Manual Does This Relate To?
- [x] Admin Manual
- [ ] Developer Manual
- [ ] User Manual
- [ ] Android
- [ ] iOS
- [ ] Branded Clients
- [ ] Desktop Client
- [ ] Other

@phil-davis fyi

**Comment by mmattel** [comment]
@phil-davis I "guess" that php7.3 support has been removed from core and will be published in 10.12, means we need to adapt the `antora.yml` file in this repo/master. Currently we have:

```
asciidoc:
  attributes:
    minimum-php-version: '7.3'
    minimum-php-printed: '7.3.0'
    minimum-php-version-short-code: '73'
    recommended-php-version: '7.4'
    recommended-php-version-short-code: '74'
    supported-php-versions: '7.3 and 7.4'
```
Can you advice?

**Comment by phil-davis** [comment]
Yes, PHP 7.3 support is dropped in the upcoming core 10.12 release.

core 10.12 must run PHP 7.4
